tikv-configuration-file.md

```diff
@@ -513,14 +513,11 @@ raftstore 相关的配置项。
 
 ### `raft-max-size-per-msg`
 
-> **注意：**
->
-> 该配置项不支持通过 SQL 语句查询，但支持在配置文件中进行配置。
-
 + 产生的单个消息包的大小限制，软限制。
 + 默认值：1MB
 + 最小值：0
-+ 单位：MB
++ 最大值: 3GB
```

Comment:
The actually max value should be 3GiB as GiB is slightly bigger than GB. But I'm not sure if introduce GiB here is proper since it may confuse user who is not familiar with the difference between GB/GiB.

Reply:
I personally agree with using "GiB" for correctness. How do you @TomShawn @ethercflow think?

Reply:
I agree with using "GiB" for correctness

Reply:
If GiB is technically more correct, what about the 46 usages of GB in this document? Shall we unify them all as GiB? Or only focus on this config?

Reply:
I think binary size(KiB/MiB/GiB.) is more accurate than decimal size (KB/MB/GB) for all of them.

Reply:
I find in tikv's implementation, the size unit is always treated as binary unit. For example, both M and MB are treated as MiB, so I think we can use KB/MB/GB in this doc as there are the most understandable.